modules/sharedaddy/sharing.php

```diff
@@ -262,7 +267,7 @@ public function management_page() {
 						<h3><?php _e( 'Live Preview', 'jetpack' ); ?></h3>
 					</td>
 					<td class="services">
-						<h2<?php if ( count( $enabled['all'] ) > 0 ) echo ' style="display: none"'; ?>><?php _e( 'Sharing is off. Add services above to enable.', 'jetpack' ); ?></h2>
+						<h2<?php if ( count( $enabled['all'] ) > 0 ) { echo ' style="display: none"';} ?>><?php _e( 'Sharing is off. Add services above to enable.', 'jetpack' ); ?></h2>
```

Comment:
Fair enough, let's convert it all into ternary operators, what do you say? Like this:
```
$selected = 'selected="selected"';
$hidden = ' style="display: none"'; ?>
...
<h2 <?php echo count( $enabled['all'] ) > 0 ? $hidden : '' ?>
...
<option<?php echo 'icon-text' === $global['button_style']' ? $selected : '' ?>
```